com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.Drawers.cs

```diff
@@ -112,6 +112,16 @@ static Func<int> SetGizmosDirty()
         }
 
         static void DrawGeneralContent(UniversalRenderPipelineSerializedLight serializedLight, Editor owner)
+        {
+            DrawGeneralContentInternal(serializedLight, owner, isInPreset: false);
+        }
+
+        internal static void DrawGeneralContentPreset(UniversalRenderPipelineSerializedLight serializedLight, Editor owner)
```

Comment:
This no longer needs to be internal with the PresetInspector being included in the class?